data/ua-update.json.in

```diff
@@ -70,5 +70,33 @@
   "google.at": "Mozilla/5.0 (Android 8.1.0; U; Mobile; rv:60.9) Gecko/60.9 SailfishBrowser/1.0",
   "apkpure.com": "Linux#Android 8.1.0",
   "zingmp3.vn": "Linux#Android 8.1.0",
-  "vk.com": "Linux; U; # "
+  "vk.com": "Linux; U; # ",
+  // JB#50424 all the ebay properties
+  "ebay.com": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
```

Comment:
As far as I see you're only dropping "Sailfish 3.0; ". Hence, you can use regexp for replacing it like:

```
"ebay.com": "Sailfish 3.0;# ",
```
I think easier to see the intended change from this. So, space after hash mark.

When changing multiple parts of the UA you need to use full UA.

Reply:
Thanks, this is nice and I've changed it.

By the way, just to highlight how nitpicky ebay is, it turns out the (on the face of it very sensible) regexp you suggested doesn't work. Why not? Because it adds two extra spaces into the UA string. It works fine like this though: `" Sailfish 3.0; # "`.

I don't understand how ebay are parsing the UA, but whatever they're doing, I don't like it.